File: C/prime_check.c
#include <stdio.h>
int main() {
	int num = 3;
	if (num < 0) printf("Entered number less than 0");
	else if (num%2 == 0){
		printf("Its not prime");
	}
	else{
		printf("Its prime");
	}
	return 0;
}

File: Python/prime_check.py
def main():
    n = int(input("Enter a number: "))

    isPrime = 1
    if n <= 1:
        isPrime = 0
    else:
        i = 2
        while i * i <= n:
            if n % i == 0:
                isPrime = 0
                break
            i += 1

    if isPrime:
        print(f"{n} is a Prime Number")
    else:
        print(f"{n} is Not a Prime Number")


if __name__ == "__main__":
    main()

